build.py

```diff
@@ -319,220 +358,196 @@ def upload_packages(packages, bucket_name=None, nightly=False):
                                 os.path.basename(p))
         else:
             name = os.path.basename(p)
-        if bucket.get_key(name) is None or nightly:
-            print "Uploading {}...".format(name)
-            sys.stdout.flush()
+        if bucket.get_key(name) is None or overwrite:
+            logging.info("Uploading file {}".format(name))
             k = Key(bucket)
             k.key = name
-            if nightly:
+            if overwrite:
                 n = k.set_contents_from_filename(p, replace=True)
             else:
                 n = k.set_contents_from_filename(p, replace=False)
             k.make_public()
         else:
-            print "!! Not uploading package {}, as it already exists.".format(p)
-    print ""
-    return 0
-
-def go_list(vendor=False, relative=False):
```

Comment:
Sorry, that was a casualty of the initial merge of the changes. It's fixed now, though.